beacon-chain/core/helpers/beacon_committee.go

```diff
@@ -296,42 +296,38 @@ func ShuffledIndices(s state.ReadOnlyBeaconState, epoch primitives.Epoch) ([]pri
 
 // UpdateCommitteeCache gets called at the beginning of every epoch to cache the committee shuffled indices
 // list with committee index and epoch number. It caches the shuffled indices for current epoch and next epoch.
```

Comment:
Small comment update. Without the range it looks like we are updating the committee cache for only one epoch and not both "current epoch and next epoch."

Reply:
Done, thanks !